source/ranges.tex

```diff
@@ -901,24 +901,24 @@
 
 \rSec2[range.view]{Views}
 
-\indexlibrary{\idxcode{View}}%
+\indexlibrary{\idxcode{view}}%
 \begin{itemdecl}
 template<class T>
-  concept View =
+  concept view =
     range<T> && semiregular<T> && enable_view<T>;
 \end{itemdecl}
 
 \begin{itemdescr}
-% FIXME: This should explicitly say when View is modeled.
+% FIXME: This should explicitly say when view is modeled.
 \pnum
-The \libconcept{View} concept specifies the requirements of a \libconcept{range} type
+The \libconcept{view} concept specifies the requirements of a \libconcept{range} type
 that has constant time copy, move, and assignment operators; that is, the cost of
 these operations is not proportional to the number of elements in the
-\tcode{View}.
+\tcode{view}.
 
 \pnum
 \begin{example}
-Examples of \tcode{View}s are:
+Examples of \tcode{view}s are:
```

Comment:
> This is a common problem throughout the library for "predicate"s, "sentinel"s, etc.

Yes, it is. And I failed to achieve consistency in [ranges], let alone the rest of the library. Maybe we should copy [my comment above](https://github.com/cplusplus/draft/pull/3149#discussion_r312314312) into an editorial issue and make a determination about whether or not it should be policy so we can enforce it library-wide rather than having it be my personal war on the library?